app/controllers/calls_controller.rb

```diff
@@ -20,17 +20,17 @@ def start
   # CallSid - default param from twilio (required)
   def new_connection
     response = Twilio::TwiML::Response.new do |r|
-      if active_call.next_target.present?
+      if active_call.next_target.present? && !active_call.exceeded_max_connections?
         connection = active_call.create_connection!
-        # r.Play AudioFileFetcher.file_for_key('connecting_local')
-        r.Play AudioFileFetcher.file_for_key('connecting_to_representative')
-        r.Play AudioFileFetcher.file_for_key('star_to_disconnect')
+        # r.Play AudioFileFetcher.audio_url_for_key('connecting_local')
+        r.Play AudioFileFetcher.audio_url_for_key('connecting_to_representative')
```

Comment:
Line is too long. [81/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.